python/cugraph/louvain/test_louvain.py

```diff
@@ -80,9 +80,9 @@ def networkx_call(M):
     return parts
 
 
-DATASETS = ['/datasets/networks/karate.mtx',
-            '/datasets/networks/dolphins.mtx',
-            '/datasets/networks/netscience.mtx']
+DATASETS = ['../datasets/karate.mtx',
+            '../datasets/dolphins.mtx',
+            '../datasets/netscience.mtx']
```

Comment:
Absolute paths are still used in this file.

```python
110 DATASETS = ['/datasets/networks/karate.mtx',
111             '/datasets/networks/dolphins.mtx']
```

Reply:
Good catch! I fixed it with ced91aa